src/models/pmv.js

```diff
@@ -0,0 +1,216 @@
+ *
+ * The PMV is an index that predicts the mean value of the thermal sensation votes (self-reported perceptions)
+ * of a large group of people on a sensation scale expressed from –3 to +3 corresponding to the categories: cold, cool,
+ * slightly cool, neutral, slightly warm, warm, and hot. [1]_
+ *
+ * While the PMV equation is the same for both the ISO and ASHRAE standards, in the ASHRAE 55 PMV equation, the SET is
+ * used to calculate the cooling effect first, this is then subtracted from both the air and mean radiant temperatures,
+ * and the differences are used as input to the PMV model, while the airspeed is set to 0.1m/s. Please read more in the
+ * Note below.
+ *
+ * [1]    ANSI, & ASHRAE. (2020). Thermal Environmental Conditions for Human Occupancy. Atlanta.
+ *
+ * [2]    ISO. (2005). ISO 7730 - Ergonomics of the thermal environment — Analytical determination and interpretation
+ *      of thermal comfort using calculation of the PMV and PPD indices and local thermal comfort criteria.
+ *
+ * This is a version that supports arrays.
+ * @see {@link pmv} for a version that supports scalar arguments.
+ *
+ * @public
+ * @memberof models
+ * @docname Predicted Mean Vote (PMV) (array version)
+ *
+ * @param {number[]} tdb - dry bulb air temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param {number[]} tr - mean radiant temperature, default in [°C] in [°F] if `units` = 'IP'
+ * @param {number[]} vr - relative air speed, default in [m/s] in [fps] if `units` = 'IP'
+ * @param {number[]} rh - relative humidity, [%]
+ * @param {number[]} met - metabolic rate, [met]
+ * @param {number[]} clo - clothing insulation, [clo]
+ * @param {number[]} wme - external work, [met] default 0
+ * @param {"ISO"|"ASHRAE"|"iso"|"ashrae"} standard - comfort standard used for calculation
+ * @param {pmvKwargs} kwargs - additional arguments
+ *
+ * @returns {number[]} pmv - Predicted Mean Vote
+ *
+ * @notes You can use this function to calculate the `PMV`_ [1]_ [2]_.
+ * _PMV:
+ * @see {@link https://en.wikipedia.org/wiki/Thermal_comfort#PMV/PPD_method}
+ * _Addendum C to Standard 55-2020:
+ * @see  {@link https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/55_2020_c_20210430.pdf}
+ *
+ * @example
+ * import {pmv_array} from "./models/pmv.js";
+ * import {v_relative_array, clo_dynamic_array} from "./utilities/utilities.js";
+ *
+ * const tdb = [22, 25];
+ * const tr = [25, 25];
+ * const rh = [50, 50];
+ * const v = [0.1, 0.1];
+ * const met = [1.4, 1.4];
+ * const clo = [0.5, 0.5];
+ *
+ * // calculate relative air speed
+ * const v_r = v_relative_array(v, met);
+ * // calculate dynamic clothing
+ * const clo_d = clo_dynamic_array(clo, met);
+ *
+ * const results = pmv_array(tdb, tr, v_r, rh, met, clo_d);
+ * console.log(results); // [-0.47, 0.06]
+ */
+export function pmv_array(
+  tdb,
+  tr,
+  vr,
+  rh,
+  met,
+  clo,
+  wme = [],
```

Comment:
you need to fix the array size, `pmv_ppd_array` should not handle different array sizes

Reply:
done